python/ray/serve/api.py

```diff
@@ -71,11 +73,15 @@ def init(name=None,
         name (str): A unique name for this serve instance. This allows
             multiple serve instances to run on the same ray cluster. Must be
             specified in all subsequent serve.init() calls.
-        http_host (str): Host for HTTP server. Default to "0.0.0.0".
-        http_port (int): Port for HTTP server. Default to 8000.
+        http_host (str): Host for HTTP servers. Default to "0.0.0.0". Serve
+            starts one HTTP server per node.
```

Comment:
```suggestion
            starts one HTTP server per node in the Ray cluster.
```

Reply:
Will it be a problem that this can't be configured per-node? I guess common case is either localhost or `0.0.0.0` but there may be special setups where this causes an issue

Reply:
One special case will be binding to the internal IP which is different per node. But 127.0.0.1 is alias to that. Can't think of any other special case for now.